code/main/media/mediaModule.brs

```diff
@@ -79,6 +81,13 @@ function _adb_MediaModule(configurationModule as object, edgeRequestQueue as obj
 
         end sub,
 
+        ''' Handles responses dispatched by the event processor
+        processResponseEvent: function(event as object) as void
+            _adb_logInfo("MediaModule::processResponseEvent() - Received response event:(" + chr(10) + FormatJson(event) + chr(10) + ")")
+
+            m._sessionManager.handleResponseEvent(event)
```

Comment:
If possible, can you add a check here so that mediaModule only handles media response?

Reply:
In the current state it will be looking at handles or MediaModule being aware of the requestIds. We could add additional property to responses like (parentModule or something like that) which can be used to filer or do some more logic while creating responses like sending each handle as a separate event. 

I would like to take this as a separate PR probably during this: #175

Reply:
We don’t need to add parentModule. Simply checking the handle will allow the media to exit early.